content/200-orm/200-prisma-client/300-client-extensions/200-extension-examples.mdx

```diff
@@ -29,6 +29,7 @@ The following is a list of extensions created by the community. If you want to c
 | [`prisma-extension-random`](https://github.com/nkeil/prisma-extension-random)                  | Lets you query for random rows in your database                                                                       |
 | [`prisma-paginate`](https://github.com/sandrewTx08/prisma-paginate)                            | Adds support for paginating read queries                                                                              |
 | [`prisma-extension-streamdal`](https://github.com/streamdal/prisma-extension-streamdal) | Adds support for Code-Native data pipelines using Streamdal |
+| [`prisma-rbac`](https://github.com/multipliedtwice/prisma-rbac) | Adds customizable role-based access control |
```

Comment:
Generally speaking we'd really prefer to keep the naming scheme of `prisma-extension-{your extension name}` but as you've already made it, I'm not gonna make you change it 😅